src/TripSegment/index.js

```diff
@@ -284,7 +315,13 @@ class TripSegment extends React.PureComponent<Props, State> {
     const { expanded, initialExpanded, contentHeight } = this.state;
 
     return (
-      <StyledTripSegment dataTest={dataTest}>
+      <StyledTripSegment
+        dataTest={dataTest}
+        tabIndex="0"
```

Comment:
When someone is gonna use this component on e.g. search, won't be the `zero` a problem? I think that we should provide a property for specifying the tabIndex - based on the document hierarchy, right?

Reply:
tabindex="0" is respecting document DOM. Can you be more specific where it could be an issue?

Reply:
I meant a case when someone wants to use custom `tabIndex` and don't use the document hierarchy. Like the `0` use default, but can be specified explicitly.

Reply:
Oh I see, yeah that would make sense, with defaulting to 0. Will Add!